subsys/bluetooth/audio/Kconfig.mctl

```diff
@@ -0,0 +1,83 @@
+# Bluetooth Audio - Media control configuration options
+
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# TODO: Decide, and add, top-level namespace prefix (currently none) to symbols.
+# Dependent upon where the module will be placed.
+
+config MCTL
+	bool "Support for media player control"
+	select EXPERIMENTAL
+	help
+	  Enables support for control of local and remote media players
+	  To enable support for control of a local media player, support for
+	  local media player must be enabled
+
+if MCTL
+
+config MCTL_LOCAL_PLAYER_CONTROL
+	bool "Support for control of local media player"
+	# TODO: Move MPL out of BT susbystem
+	depends on BT_MPL
+	help
+	  This option enables support for control of a local media player
+
+config MCTL_LOCAL_PLAYER_LOCAL_CONTROL
+	bool "Support for local control of local media player"
+	depends on MCTL_LOCAL_PLAYER_CONTROL
+	help
+	  This option enables support for local application control of local
+	   media players
+
+config MCTL_LOCAL_PLAYER_REMOTE_CONTROL
+	bool "Support for remote control of local media players"
+	depends on MCTL_LOCAL_PLAYER_CONTROL
+	help
+	  This option enables support for remote control of local media
+	   players. Currently, the only option for remote control is via
+	   Bluetooth.  More options may be added later.
+
+config MCTL_LOCAL_PLAYER_REMOTE_CONTROL_BT
+	bool "Support for remote control of local media players via Bluetooth"
+	depends on MCTL_LOCAL_PLAYER_REMOTE_CONTROL
+	depends on BT_MCS
+	help
+	  This option enables support for remote control of local media
+	   players, using GATT and the Media Control Service
+
+config MCTL_REMOTE_PLAYER_CONTROL
+	bool "Support for control of remote media player"
+	default y
+	help
+	  This options enables support for control of a remote media player.
+	  Currently, the only option for remote player control is via Bluetooth.
+	  More options may be added later.
+
+config MCTL_REMOTE_PLAYER_CONTROL_BT
+	bool "Support for control of remote media player via Bluetooth"
+	default y
+	depends on MCTL_REMOTE_PLAYER_CONTROL
+	depends on BT_MCC
+	help
+	  This options enables support for control of a remote media player,
+	  using GATT and the Media Control Client
+
+config MCTL_REMOTE_PLAYER_CONTROL_BT_OBJECTS
+	bool "Support for accessing objects on remote player"
+	depends on MCTL_REMOTE_PLAYER_CONTROL_BT
+	default y
+	depends on BT_MCC_OTS
+	help
+	  This options enables support for accessing objects on a remote media
+	  player
```

Comment:
It feels like there is still some miscommunication. My take from the meeting was that generic (i.e. non-BT specific) features/subsystems should have have any BT specific dependencies nor config options. 

Furthermore this config option feels too generic: What is an "object" in a media controller?

Reply:
Clarified the objects.